ledger/sandbox/src/main/scala/com/digitalasset/platform/store/dao/events/EventsRange.scala

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) 2020 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+package com.daml.platform.store.dao.events
+
+import anorm.SqlParser.get
+import anorm.{RowParser, SqlStringInterpolation}
+import com.daml.ledger.participant.state.v1.Offset
+import scalaz.syntax.std.option._
+
+final case class EventsRange[A](startExclusive: A, endInclusive: A)
+
+object EventsRange {
+  private val EmptyRowIdRange = EventsRange(0L, 0L)
+
+  private implicit val `offset to statement converter` =
+    com.daml.platform.store.Conversions.OffsetToStatement
```

Comment:
Why not simply importing `com.daml.platform.store.Conversions.OffsetToStatement`?

Reply:
yeah, can be done... it is just idea was auto-removing it every time I was optmizing imports, so I put it into a var.... but sure I will move it into an import